src/module/dy/create-model.js

```diff
@@ -77,10 +77,10 @@ module.exports = (kwargs) => {
       }))
     }),
     BillingMode: 'PAY_PER_REQUEST'
-  };
+  });
 
   return {
-    schema,
+    schema: genSchema(),
```

Comment:
This is not the best. Are we just going to pass the handler all the way through? If we do I need to write some more tests for coverage

Reply:
I'm not sure I understand?

Reply:
pass genSchema() through everywhere?